hail/src/main/scala/is/hail/HailContext.scala

```diff
@@ -298,6 +304,46 @@ object HailContext {
     hc
   }
 
+  def createDistributed(hostname: String,
+    logFile: String = "hail.log",
+    quiet: Boolean = false,
+    append: Boolean = false,
+    minBlockSize: Long = 1L,
+    branchingFactor: Int = 50,
+    tmpDir: String = "/tmp",
+    optimizerIterations: Int = 3): HailContext = contextLock.synchronized {
+    require(theContext == null)
+    val sConf = createSparkConf("Hail", None, "local[*]", minBlockSize)
+    val hConf = SparkHadoopUtil.get.newConfiguration(sConf)
```

Comment:
I think you can just do `val hConf = new hadoop.conf.Configuration()`.  You'll miss some s3 settings and the Hadoop settings set via `spark.hadoop...` but we shouldn't be setting those in the distributed setting.